Sprites

AmandaEkman opened the pull request:
Har lagt in PSD filer i sprites i from av enemy ship, player ship samt stjärnor. Detta finns i mappen sprites

AmandaEkman committed:
Sprites

Har lagt in PSD filer i sprites i from av enemy ship, player ship samt stjärnor. Detta finns i mappen sprites

ScrubbyAlien commented on ProjectSettings/ProjectSettings.asset:
Har du gjort någon ändring i BuildSettings? Om inte så kan du ta bort alla ändringar här, annars skicka ett litet meddelande på discord så reder vi ut det

ScrubbyAlien commented on .vsconfig:
den här filen behöver inte vara med i git

ScrubbyAlien reviewed (changes requested):
skulle du kunna lägga till png filer istället för psd-filer? eller använde du psd av nån särskild anledning?

jag vet inte om det spelar jättestor roll men i regel så undvik mellanslag i filnamn, det blir lättare att felsöka om något går fel då. Vanligt är att man bara använder underscore "_" istället, alltså enemy_ship och stjärnor_prototyp

se kommentarer på .vsconfig och ProjectSettings.asset också

ScrubbyAlien committed:
Merge branch 'main' into Sprites

AmandaEkman committed:
Tagit bort dessa filer

Har tagit bort ändringarna på projectsettings.assets och .vsconfig

AmandaEkman committed:
Merge branch 'Sprites' of https://github.com/DSU2024-Grupp12/Spelprojekt1 into Sprites

ScrubbyAlien reviewed (changes requested):
ändra så filnamnen följer stilguiden

AmandaEkman committed:
Sprites

Har lagt in och klippt sprites. Så att vi har något att leka med och testa. Som ni ser så finns det 4 st resurser, detta är för att de har lika olika textur och kunde inte avgöra vad som passar spelet bäst.

AmandaEkman committed:
Bytte namn

Bytte namn på diverse sprites
